Assets/Standard Assets/Characters/FirstPersonCharacter/Scripts/FirstPersonController.cs

```diff
@@ -47,6 +50,26 @@ public class FirstPersonController : MonoBehaviour
         [System.NonSerialized]
         public bool isStunned;
 
+        public GameObject FindClosestKeyLocation()
+        {
+            GameObject[] gos;
+            gos = GameObject.FindGameObjectsWithTag("KeyLocation");
```

Comment:
Please no variable namings with 2-3 letters unless it's general term to use (like DDA). Here you can call it `keyLocations`

Reply:
This applies to some later variables in this function as well

Reply:
Will need to be remade anyway. Leave it for now